test/test_integration.py

```diff
@@ -1078,8 +1126,6 @@ def socket_handler(listener):
             d.data = b'1234567890' * 2
             d.flags.add('END_STREAM')
             sock.send(d.serialize())
-
-            send_event.wait(5)
```

Comment:
I don't think this really answered what the reason was for removing this. Was it causing a problem?